.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
+
+[{*.gradle.kts,*.kt,*.kts,*.main.kts,*.space.kts}]
+max_line_length = 1500
```

Comment:
Usikker på om veldig lange linjer vil øke lesbarheten – da blir det vel mye horisontal scrolling?
Forøvrig inneholder min editorconfig mye mer enn dette, usikker på hva som da vil skje med de andre innstillingene.